src/HttpClient/FileGetContentsClient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Voronkovich\SberbankAcquiring\HttpClient;
+
+/**
+ * Simple HTTP client using file_get_contents() with stream_context_create().
+ *
+ * @author     disjointed
+ */
+class FileGetContentsClient implements HttpClientInterface
+{
+	public function request($uri, $method = 'GET', array $headers = array(), array $data = array())
+	{
+		$response = file_get_contents($uri, false, stream_context_create([
+			'http' => [
+				'method' => $method,
+				'header' => $headers,
+				'content' => http_build_query($data),
+				'host' => 'securepayments.sberbank.ru',
```

Comment:
There is no such option in the official documentation: http://php.net/manual/en/context.http.php#refsect1-context.http-options